server/loader.py

```diff
@@ -23,4 +19,4 @@ def load(info):
                                                      info['serverRoot'])
     info['serverRoot'].api = info['serverRoot'].girder.api
 
-    info['apiRoot'].video = track.TrackResource()
\ No newline at end of file
+    info['apiRoot'].video = track.TrackResource()
```

Comment:
*Please* tell me this means lint enforces newline at end of tile? 🙏

Reply:
@mwoehlke-kitware, Yes, the linting enforces newline at the end of files. 

Do you know how to run tests? If you don't, you could try with this girder [documentation](https://girder.readthedocs.io/en/latest/development.html#running-the-tests).